.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 /phpunit.xml
 .phpunit.result.cache
+.idea
```

Comment:
Can you add this to your global gitignore?

Reply:
Resolved 🎉